apps/speed.c

```diff
@@ -652,13 +650,14 @@ static int MD5_loop(void *args)
         MD5(buf, lengths[testnum], md5);
     return count;
 }
+#endif
 
 static int HMAC_loop(void *args)
 {
     loopargs_t *tempargs = (loopargs_t *)args;
     unsigned char *buf = tempargs->buf;
     HMAC_CTX *hctx = tempargs->hctx;
-    unsigned char hmac[MD5_DIGEST_LENGTH];
+    unsigned char hmac[SHA_DIGEST_LENGTH];
```

Comment:
I'm not sure about this change. The problem is that by doing this you cannot compare the performance of HMAC across different OpenSSL versions.